ru.toxblh.KeeneticManager.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id" : "ru.toxblh.KeeneticManager",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "keeneticmanager",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=xdg-pictures",
+        "--talk-name=org.freedesktop.Notifications",
```

Comment:
is notifications needed? It should use the portal.

Reply:
no, removed

Reply:
that fixed as well